build_tools/packaging/linux/build_package.py

```diff
@@ -786,7 +912,15 @@ def main(argv: list[str]):
         required=True,
         help="Choose the package format to be generated: DEB or RPM",
     )
-    p.add_argument("--rocm-version", default="9.9.9", help="ROCm Release version")
+    p.add_argument(
+        "--run-id",
+        type=str,
```

Comment:
Required?

Reply:
Currently in ROCM mainline, build number is also part of the package name.
Not sure whether we can use "run-id"  in ROCk package name. Currently its not used. 
Can remove it in next revision